### DISCUSSION TITLE
Enlarge the taskbar size and the icon size

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#543] [General]
Is there a possibility to enlarge the taskbar size and the icon size as in Windows 10? Or do I use the special registry settings in Windows 11?

**Comment by valinet** [comment]
Have you read this? https://github.com/valinet/ExplorerPatcher/issues/56

**Comment by ghost** [comment]
I did now. Sorry.

Johan Wouters
________________________________

**Comment by valinet** [comment]
Right click taskbar - Properties - Taskbar section - Taskbar icon size.
